DXMainClient/DXGUI/Generic/MainMenu.cs

```diff
@@ -696,7 +694,30 @@ private void BtnOptions_LeftClick(object sender, EventArgs e)
 
         private void BtnNewCampaign_LeftClick(object sender, EventArgs e)
         {
-            innerPanel.Show(innerPanel.CampaignSelector);
+            if (ClientConfiguration.Instance.LocalGame == ProgramConstants.YR)
```

Comment:
While saving a constant for this could be handy, the rest of the client has `#if YR` conditional compilation symbol checks so I'd prefer that for consistency.